GitHub/GitHub_Create_Pull_Request.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "621d75d9-b4ce-4126-bcd9-8bae98b3c5a1",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1a5e34d-b4ae-4163-851b-10c6db965fc6",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "# GitHub - Create Pull Request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ee32fea-d721-446e-ab75-74ca1bfb1210",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #github #pygithub #pullrequest #create #assign #issue"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11aa93e7-943c-4a70-804f-03ee199f1024",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [Florent Ravenel](https://www.linkedin.com/in/florent-ravenel/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae0b030c-6330-4622-9cb3-7316923b6bbd",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Description:** This notebook will show how to create a pull request with assignee and link to issue using pygithub library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f49d9417-cbb8-4c68-9b13-7ccb746c076d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "<u>References:</u>\n",
+    "- [PyGithub Documentation](https://pygithub.readthedocs.io/en/latest/)\n",
+    "- [GitHub Documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "314a3e2b-68cf-4ed8-af3e-83fc52a42590",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "438e3250-f55d-4f66-8233-d18832c55369",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55b6723a-6d1d-42c7-a1bd-764d9c3b4333",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import github"
```

Comment:
Better to have a try-except block to overcome the `ModuleNotFoundError`

Reply:
Sure!